scripts/build.js

```diff
@@ -86,13 +86,11 @@ if (POSTINSTALL_BUILD_CWD !== CWD) {
           })
           .value()
           .join(' ');
-
-        Promise.try(function() {
-          if (!_.isEmpty(installArgs)) {
-            console.log('Installing dependencies');
-            return exec('npm install ' + installArgs, opts);
-          }
-        })
+        var needsDepInstallation = !_.isEmpty(installArgs);
+        var dependenciesInstalledQ = needsDepInstallation
+          ? exec('npm install ' + installArgs, opts)
+          : Promise.resolve();
+        dependenciesInstalledQ
```

Comment:
does this line do anything?

Reply:
yeah there is a `.then(` on the line below